### ISSUE TITLE
Doesn't work with GIMP 2.99.18

### ISSUE THREAD

**Issue by rempas** [Uzugijin/GIMP-IconThemes#2]
It just shows the default icons instead.

**Comment by Uzugijin** [comment]
Hey @rempas ! Thanks for reporting!

Do you have this option unchecked in Preferences > Interface > Icon Themes?
![image](https://github.com/user-attachments/assets/eee06950-58b6-42f1-b058-4c718acc5fb9)

Up until now the icon packs were registered as 'color' and their 'symbolic' counterpart didn't exist so if that option is checked for you, then it will default to GIMP symbolic (greyscale icons).

You can also try this [pre-release](https://github.com/Uzugijin/GIMP-IconThemes/releases/tag/v2) where it's not an issue anymore. At least it shouldn't be. I haven't tested on 2.99.18 since it has some breaking changes that i didn't want to deal until GIMP 3 RC1 release and as far as I know they didn't change how icons are read by the software but if it doesn't work for you, I'll install and check.

Also, if you rather would use a custom folder for the icon pack to live in, then makes sure said folder has a hicolor folder with at an empty index.theme file. can be created with text editor and renamed.

![image](https://github.com/user-attachments/assets/f0d99a3e-91da-4550-b228-d19934e8a411)

Hope it helps!

**Comment by rempas** [comment]
Thanks for the reply! I unchecked the symbolic icons anyway as I've seen the instructions. Now, for the directory, I tried to use one of the two that GIMP uses. I am using Arch Linux so, maybe on OS problem? Or, maybe GIMP has a bug with icons.

But you know what. Just forget it! Don't work more! GIMP 3.0 RC will be out soon, so, no reason to bother with the last development version. I'll remember to check the icons 5-6 days after RC1 releases so, you have plenty of time to test without any pressure ;)
